code/modules/client/preferences.dm

```diff
@@ -661,18 +658,22 @@ GLOBAL_LIST_EMPTY(chosen_names)
 	dat += "<td width='33%' align='left'></td>"
 	dat += "<td width='33%' align='center'>"
 	dat += "<a href='?_src_=prefs;preference=bespecial'><b>[next_special_trait ? "<font color='red'>SPECIAL</font>" : "Be Special"]</b></a><BR>"
-	if(SSticker.current_state <= GAME_STATE_PREGAME)
-		switch(N.ready)
```

Comment:
For anyone looking at this- the only actual changes here are this and the preceding line, and last two line of this block(675 & 676). The rest is indented one level deeper than before because it's now inside an if(). This is fine.

Reply:
Honestly this should be factored out into its own function for readability, I think.